support/nginx/peertube

```diff
@@ -57,6 +57,16 @@ server {
     root /var/www/certbot;
   }
 
+  # Bypass PeerTube for performance reasons. Could be removed
+  # Should be consistent with client-overrides assets list in /server/controllers/client.ts
+  location ~ ^/client/(assets/images/(icons/icon-36x36\.png|icons/icon-48x48\.png|icons/icon-72x72\.png|icons/icon-96x96\.png|icons/icon-144x144\.png|icons/icon-192x192\.png|icons/icon-512x512\.png|logo\.svg|favicon\.png))$ {
+    add_header Cache-Control "public, max-age=31536000, immutable";
+
+    root /var/www/peertube;
+
+    try_files /storage/client-overrides/$1 /peertube-latest/client/dist/$1 $uri;
```

Comment:
```suggestion
    root /var/www/peertube/storage/client-overrides;

    try_files /$1 $uri;
```

The next location block to match will be the client assets location block, which already does this.